Modelica/Utilities/Strings.mo

```diff
@@ -172,6 +172,50 @@ isEmpty(\"a\");      // returns false
 </html>"));
   end isEmpty;
 
+  function contains "Check if a string contains a substring"
+
+    extends Modelica.Icons.Function;
+
+    input String string "String to check";
+    input String searchString "Substring to look for";
+    input Boolean caseSensitive=true
+      "= false, if lower and upper case are ignored for the search";
+    output Boolean result "= true, if searchString can be found anywhere in string";
+
+  algorithm
+    if length(searchString) > 0 then
+      result := find(string, searchString, 1, caseSensitive) > 0;
+    else
+      result := string == searchString;
```

Comment:
We might define the case with empty searchString to always return `true` (as e.g., Python does).

Reply:
Done with 6954f65 - Return true for empty search strings of Strings.contains